### PR TITLE
1.1.0

### PR DESCRIPTION
**Pull request by jhrabi**
Want to create 1.1.0 unless there are any objections. The pull functionality would be especially useful.

### CODE REVIEW

File: nimbusapp [hunk 1]
```diff
@@ -140,30 +204,68 @@ if [[ -z ${PROJECT} ]]; then
...
+          if [[ -s "${COMPOSE_FILE}" ]]; then
+            mv -v "${COMPOSE_FILE}" "${COMPOSE_BACKUP}"
+          else
+            rm -vf "${COMPOSE_FILE}"
+          fi
+       fi # action == render
+
+       mkdir -p "${COMPOSE_DIR}"
+
+       echo
```
Comment (jhrabi) [inline nimbusapp:230]: Is there a specific reason for the extra echo?

File: nimbusapp [hunk 2]
```diff
@@ -140,30 +204,68 @@ if [[ -z ${PROJECT} ]]; then
...
-fi
+COMPOSE_ARGS="$@"
+COMPOSE_DIR="${HOME}/.nimbusapp/cache/${PROJECT}/${REPOSITORY}/${IMAGE}/${VERSION}"
+COMPOSE_FILE="${COMPOSE_DIR}/${IMAGE}.yml"
+COMPOSE_BACKUP="${COMPOSE_FILE}.bk"
 
-COMMAND="docker-app ${DOCKERAPP_ACTION}"
+if [[ -n "${DOCKERAPP_ACTION}" ]]; then
+   if curl -s --connect-timeout "${NIMBUS_DOCKERHUB_TIMEOUT}" "${NIMBUS_DOCKERHUB_URL}" > /dev/null 2>&1; then
+       DOCKERAPP_COMMAND="${DOCKERAPP_COMMAND} ${DOCKERAPP_ACTION} ${DOCKERAPP_ARGS} ${REPOSITORY}/${IMAGE}:${VERSION}"
```
Comment (jhrabi) [inline nimbusapp:214]: This seems to be resulting in an extra space for the render command. Output looks like this:
Running: docker-app render  admpresales/devops:1.1.7.0 > /home/demo/.nimbusapp/cache/devops/admpresales/devops/1.1.7.0/devops.yml

File: nimbusapp [hunk 3]
```diff
@@ -140,30 +204,68 @@ if [[ -z ${PROJECT} ]]; then
...
+       fi
+   fi # curl check
 fi
 
-echo -e "Running ${COMMAND}"
-eval ${COMMAND}
\ No newline at end of file
+if [[ -n "${COMPOSE_ACTION}" ]]; then
+  COMPOSE_COMMAND="${COMPOSE_COMMAND} -p ${PROJECT} -f ${COMPOSE_FILE} ${COMPOSE_ACTION} ${COMPOSE_ARGS}"
+  echo "Running: ${COMPOSE_COMMAND}"
```
Comment (jhrabi) [inline nimbusapp:269]: We might need to add verbose messaging as it's starting to look spammy with the extra message talking about the caching. Thoughts?

Comment (jasoncorlett) [reply nimbusapp:269]: Agreed, I'll see about adding something, in most cases I think all the user will want is the output of the docker-compose

File: nimbusapp (hunk 2 above)
Comment (jasoncorlett) [reply nimbusapp:214]: I think those are the spaces around the `${DOCKERAPP_ARGS}`, which is empty in the example

File: nimbusapp (hunk 3 above)
Comment (jasoncorlett) [reply nimbusapp:269]: Agreed, I think for most cases, users will only really need the output of `docker-compose`, I'll see about adding some verbosity settings

File: nimbusapp (hunk 1 above)
Comment (jasoncorlett) [reply nimbusapp:230]: Spacing of the output, I'll remove that when I look at settings for verbose output

File: nimbusapp (hunk 2 above)
Comment (jhrabi) [reply nimbusapp:214]: Meh, let's not worry too much about this one, especially if we're gonna put verbose logging in.

**Comment by jhrabi** [comment]
Closing this pull request as it's out of date.

**Pull request closed without merging**